software/firmware/europi.py

```diff
@@ -254,13 +271,13 @@ def clock_division(self):
     would only return values which go up in steps of 2.
     """
 
-    def __init__(self, pin):
-        super().__init__(pin)
+    def __init__(self, pin, deadzone=0.01):
+        super().__init__(pin, deadzone=deadzone)
 
-    def percent(self, samples=None):
+    def percent(self, samples=None, deadzone=None):
         """Return the knob's position as relative percentage."""
         # Reverse range to provide increasing range.
-        return 1 - (self._sample_adc(samples) / MAX_UINT16)
+        return 1.0 - super().percent(samples, deadzone)
 
     def read_position(self, steps=100, samples=None):

```

Comment:
[required] `read_position()` is affected by the deadzone member variable (see changes to `test_read_position()`), so it should probably take the override parameter like the other methods do and pass it to `range()`.